packages/pocl-test-file-generator/index.js

```diff
@@ -0,0 +1,16 @@
+import poclTestFileGenerator from 'commander'
```

Comment:
Just a thought - would this be a good place to try out a lambda function (triggered via the gateway)  not suggesting doing it now, but rather a candidate for a future PoC

Reply:
Yeah, this package would probably be a decent candidate for a lambda function